src/test/java/org/jabref/logic/journals/JournalAbbreviationRepositoryTest.java

```diff
@@ -17,7 +17,7 @@ public void empty() {
     @Test
     public void oneElement() {
         JournalAbbreviationRepository repository = new JournalAbbreviationRepository();
-        repository.addEntry(new Abbreviation("Long Name", "L. N."));
+        repository.addEntry(new Abbreviation("Long Name", "L. N.", "LN"));
```

Comment:
Please do not modify existing tests. The old ones should still work. There might be abbrevitations wihtout shortest unique. They should work as before your change.

Add new tests. No worries about duplicated code.